packages/manager/apps/dedicated/client/app/ip/components/alerts/alerts.controller.js

```diff
@@ -0,0 +1,90 @@
+export default class IpComponentsAlertsController {
+  /* @ngInject */
+  constructor($http, $q, Ip) {
+    this.$http = $http;
+    this.$q = $q;
+    this.Ip = Ip;
+    this.reset();
+  }
+
+  get messageType() {
+    if (this.error) {
+      return 'error';
+    }
+    if (this.alerts && !this.countAlerts()) {
+      return 'success';
+    }
+    return 'warning';
+  }
+
+  get serviceTypeSuffix() {
+    return `_w${this.serviceType ? '' : 'o'}_service_type`;
```

Comment:
**question:** What does `w` and `o` suggest here? Can we avoid abbreviations and use full name instead?

Reply:
Actually this is more like "w" and "w/o" that stands for "with" and "without"
Fixed :+1: